README.md

```diff
@@ -39,6 +39,12 @@ brew install flog
 
 Download gzip file from [Github Releases](https://github.com/mingrammer/flog/releases/latest) according to your OS. Then, copy the unzipped executable to under system path.
 
+### Using Docker
```

Comment:
Please use `docker` instead of `Docker`. And could you add link of official docker homepage (https://www.docker.com/) to word `docker`?

Thanks.